### PR TITLE
LPS-127936 Check if searchControl's select exists

### PR DESCRIPTION
**Pull request by julien**
In the metal+soy version of the ManagamentToolbar, [we used to check if
the searchContainer's select (plugin) was present](https://github.com/liferay/liferay-portal/blob/eb44efa36ce0ed22e8d59a1a2acce659a6f411e2/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.es.js#L55-L60) before trying to use it.

In our current React version, we [don't have this check](https://github.com/liferay/liferay-portal/blob/eb44efa36ce0ed22e8d59a1a2acce659a6f411e2/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js#L112-L114), and if the
`liferay-search-container-select` AUI plugin/module is not present, (because it
[doesn't always get loaded](https://github.com/liferay/liferay-portal/blob/eb44efa36ce0ed22e8d59a1a2acce659a6f411e2/portal-web/docroot/html/taglib/ui/search_iterator/lexicon/bottom.jspf#L42
)) the component will throw an error making the
management toolbar unusable.

So in this change, we're simply adding that check.

### CODE REVIEW

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js [hunk 1]
```diff
@@ -73,27 +73,29 @@ function ManagementToolbar({
...
-						onClearButtonClick={onClearSelectionButtonClick}
-						onSelectAllButtonClick={onSelectAllButtonClick}
-						searchContainerId={searchContainerId}
-						selectAllURL={selectAllURL}
-						selectable={selectable}
-						setActionDropdownItems={setActionDropdownItems}
-						setActive={setActive}
-						supportsBulkActions={supportsBulkActions}
-					/>
+					{selectable && (
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js:76]: With this check here, we no longer need to pass `selectable` prop to `<SelectionControls>` and can remove [the `{selectable && (...)}` check in component](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js#L128).

**Review by markocikos — COMMENTED** [review]
@julien A minor tweak in comment, looks good otherwise!

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js (hunk 1 above)
Comment (julien) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js:76]: @markocikos indeed. Thanks for the review.

Comment (julien) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js:76]: Updated in [aea295f](https://github.com/liferay-frontend/liferay-portal/pull/797/commits/aea295f62e0a463a37ef902a9402458e63d8abf3)

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js [hunk 2]
```diff
@@ -73,27 +73,29 @@ function ManagementToolbar({
...
+								initialSelectAllButtonVisible
+							}
+							initialSelectedItems={initialSelectedItems}
+							itemsTotal={itemsTotal}
+							onCheckboxChange={onCheckboxChange}
+							onClearButtonClick={onClearSelectionButtonClick}
+							onSelectAllButtonClick={onSelectAllButtonClick}
+							searchContainerId={searchContainerId}
+							selectAllURL={selectAllURL}
+							selectable={selectable}
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js:93]: @julien We can remove it here as well

Comment (julien) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js:93]: ah yeah sorry, I missed that one

**Review by markocikos — APPROVED** [review]

**Comment by markocikos** [comment]
@julien all good, feel free to forward! Not sure if you want to squash before.

**Comment by julien** [comment]
@markocikos thanks, yes you read my mind, didn't want to squash until we had seen this together but will do before forwarding: :+1:

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**